lib/matplotlib/figure.py

```diff
@@ -48,6 +49,8 @@
                                    TransformedBbox)
 from matplotlib.backend_bases import NonGuiException
 
+_log = logging.getLogger(__name__)
```

Comment:
can do without? (and without the import)

Reply:
I stuck it in there because I have a PR that will need it anyways ;-)  I actually think this should be at the top of most modules so folks use the debugging.  I appreciate there may be a small performance hit on some intensive modules, but hopefully not a big deal